### DISCUSSION TITLE
Neoforge compatibility?

### DISCUSSION THREAD

**Discussion by randomguy129305810481** [Sinytra/ForgifiedFabricAPI#50] [Ideas]
I just learned about existence of this mod and I think that mod concept is really interesting. However as I am shifting to using neoforge, I was wondering whether you would be able/willing to make this mod compatible with neoforge or perhaps create version for neoforge.

**Comment by RoomAlex68** [comment]
They have stated that Sinytra Connector will be porting to NeoForge 1.20.4 and Beyond.
